main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	_ "github.com/jackc/pgx/v5/stdlib"
+	"github.com/julienschmidt/httprouter"
+)
+
+// Use global database for now, this should be changed later
```

Comment:
```suggestion
// FIXME Use global database for now, this should be changed later
```

I encourage adding explicit FIXME's or TODO's since most editors pick up on these and make them plainly obvious to go back and change.